server/start-server.sh

```diff
@@ -4,9 +4,10 @@ set -o pipefail # if any code doesn't return 0, exit the script
 
 
 function setup_server() {
-  python manage.py makemigrations
-  python manage.py migrate
-  echo "from django.contrib.auth.models import User; User.objects.filter(email='admin@example.com').delete(); User.objects.create_superuser('admin', 'admin@example.com', 'nimda')" | python manage.py shell
+  source venv/bin/activate
+  python server/manage.py makemigrations
```

Comment:
This doesn't look right, as this script is already within the server folder @makindemayowa 

It should just be ` python manage.py makemigrations`

Reply:
I thought so too, but the script doesn't run without referencing the server folder.

This happens with `python manage.py makemigrations`
<img width="607" alt="screen shot 2018-09-13 at 9 46 41 pm" src="https://user-images.githubusercontent.com/25981851/45514977-cdc8ba00-b79e-11e8-9fef-e42ab51f5754.png">

This happens with `python server/manage.py makemigrations`
<img width="726" alt="screen shot 2018-09-13 at 9 47 27 pm" src="https://user-images.githubusercontent.com/25981851/45515070-0a94b100-b79f-11e8-92e6-09b89e779480.png">

Reply:
Why are you calling the `source` operation on the file?

What you should check is if this affects docker usage(it does) as docker makes use of this, that's why you should I asked you to take a second look at it.